### PR TITLE
The basic setup for jobs

### PR DESCRIPTION
**Pull request by tiffz**
Jobs aren't functional yet, but this sets up some of the basic data for them.

### CODE REVIEW

File: src/components/SayluaView/Sidebar/SpriteJobView.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const mapStateToProps = state => ({
+  activeJobsWithSprites: activeJobsWithSpritesSelector(state),
+});
+
+function SpriteJobsView(props) {
+  return (
+    <div id="user-info-section" className="sidebar-section">
+      {
+        props.activeJobsWithSprites.map(jobEntry => (
```
Comment (LikeMyBread) [inline src/components/SayluaView/Sidebar/SpriteJobView.jsx:14]: Add unique keys

File: src/modules/Den/SpriteProfile/Main.jsx [hunk 2]
```diff
@@ -165,6 +171,20 @@ class SpriteProfile extends Component {
...
         </Modal>
+
+        <Modal
+          onClose={() => { this.setState({ editingJob: false }); }}
+          opened={this.state.editingJob}
+        >
+          <h2>What will { sprite.name } do?</h2>
+          {
+            jobsList.map(job => (
+              <Button onClick={this.props.assignJob(sprite.id, job.id)}>
```
Comment (LikeMyBread) [inline src/modules/Den/SpriteProfile/Main.jsx]: Another place for keys

**Review by LikeMyBread — APPROVED** [review]
Looks good, just could use some keys.

File: src/components/SayluaView/Sidebar/SpriteJobView.jsx (hunk 1 above)
Comment (tiffz) [reply src/components/SayluaView/Sidebar/SpriteJobView.jsx:14]: Done! Thanks for the reminder; I've been bad about using keys.

File: src/modules/Den/SpriteProfile/Main.jsx (hunk 2 above)
Comment (tiffz) [reply src/modules/Den/SpriteProfile/Main.jsx]: Done!

**Pull request merged**